dvc/remote/config.py

```diff
@@ -134,6 +133,8 @@ def remove(self, name, level=None):
                 break
 
     def modify(self, name, option, value, level=None):
+        self.get_settings(name)
```

Comment:
Using `get_settings` raises the following error when a remote is not found:

```python
raise ConfigError("unable to find remote section '{name}''")
```

Reply:
@mroutis Let's add a comment there though, please.